lib/opam/opam_make_opam.mli

```diff
@@ -0,0 +1,19 @@
+open Core.Std
+open Async.Std
+
+(**
+ * This command creates an `opam` and `META` file for opam
+*)
+
+val name: String.t
+val command: Command.t
+
+val desc: String.t * Command.t
+
+val write_opam: ?name:String.t -> ?semver:String.t
+  -> target_dir:String.t
+  -> license:String.t -> maintainer:String.t Option.t -> author:String.t Option.t
+  -> homepage:String.t Option.t -> bug_reports:String.t Option.t -> dev_repo:String.t
```

Comment:
thats what it is called in the opam file. I tried to match these to those names.